packages/react-components/react-list-preview/src/components/ListItem/useListItemStyles.styles.ts

```diff
@@ -1,33 +1,69 @@
-import { makeStyles, mergeClasses } from '@griffel/react';
 import type { SlotClassNames } from '@fluentui/react-utilities';
+import { makeStyles, makeResetStyles, mergeClasses, shorthands } from '@griffel/react';
+import { createCustomFocusIndicatorStyle } from '@fluentui/react-tabster';
 import type { ListItemSlots, ListItemState } from './ListItem.types';
+import { tokens } from '@fluentui/react-theme';
 
 export const listItemClassNames: SlotClassNames<ListItemSlots> = {
   root: 'fui-ListItem',
-  // TODO: add class names for all slots on ListItemSlots.
-  // Should be of the form `<slotName>: 'fui-ListItem__<slotName>`
+  checkmark: 'fui-ListItem__checkmark',
 };
 
+const useRootBaseStyles = makeResetStyles({
+  padding: 0,
+  margin: 0,
+  textIndent: 0,
+  listStyleType: 'none',
+  ...createCustomFocusIndicatorStyle(
+    {
+      ...shorthands.outline(tokens.strokeWidthThick, 'solid', tokens.colorStrokeFocus2),
+      ...shorthands.borderRadius(tokens.borderRadiusMedium),
+    },
+    { selector: 'focus' },
+  ),
+});
+
+const useCheckmarkBaseStyles = makeStyles({
+  root: {
+    alignSelf: 'center',
+    //eslint-disable-next-line
+    '& .fui-Checkbox__indicator': {
```

Comment:
You should be able to do:

```ts
const useCheckmarkBaseStyles = makeStyles({
  indicator: {}
})
```

&

```ts
  if (state.checkmark) {
    state.checkmark.className = mergeClasses(
      listItemClassNames.checkmark,
      checkmarkBaseStyles.root,
      state.checkmark?.className,
    );

     state.checkmark.indicator ??= {}
    state.checkmark.className = mergeClasses(
      checkmarkBaseStyles.indicator,
      state.checkmark.indicator.className,
    );  
  }
```

Reply:
Indicator is a slot, unfortunately this doesn't work. We agreed to keep this as is and resolve the comment.